goreleaser/docker.yml

```diff
@@ -47,6 +47,7 @@ dockers:
       - go.mod
       - go.sum
       - src
+      - web
```

Comment:
I know this is superficial, but would it make sense to call this like `internal_dex.config` or something?